tslint.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
These are the only enforced rules. For a first pass, I think it makes sense to keep the rule set consistent with other Microsoft OSS TypeScript projects but we can add additional rules in the future.